docs/acceptance.md

```diff
@@ -1,41 +1,28 @@
-<img width="840px" height="auto" src="./acceptance-wires.png" alt="Splitting basics">
+# Shield Acceptance
 
-### In General
-- [ ] It should generally match Photon styles.
-- [ ] It should have a way for users to provide feedback without leaving the experiment. See #6
+[Shield Study Guide](https://docs.google.com/document/d/1KDJPrWmSclap7HSqjDoN9cGud4WzB9i8wxlguYPpGkg/edit)
 
-### Context Clicks
-- [x] It should register a _link_ context menu item for opening sites in the sidebar. See #3
-- [x] It should register a _page_ and _tab_ context menu item for opening sites in the sidebar. See #4
-- [ ] It should register a _bookmarks_ context menu item for opening sites from the bookmark toolbar.
-- [ ] Both context menu buttons should include the extension icon.
+## General
+- [ ] It should more or less match photon styles
+- [ ] It should fit with both default and dark Firefox themes
+- [ ] It should remove links to the Test Pilot survey
 
-### SV Homepage
-- [ ] It should have a homepage
-- [ ] It should have a feedback button that links to an external survey with the query params `ver={add-on-version}`, `ref=sidebar` and `rel={firefox-release}`. See #6
-- [ ] It should link to onboarding content.
-- [ ] It should include the extension icon in the header.
+## Metrics
+- [ ] It should not send any metrics reporting to GA
+- [ ] All current metrics should be adapted to send to Telemetry
+- [ ] It should instrument a new `panel_visible_today` metric that is reported daily as a boolean based on whether the user had any web content visible in Side View on a given day
+- [ ] It should instrument a new `total_daily_uri_to_sv` metric which counts the number of new URIs sent to Side View each day. Note: even if this is 0 for a given day, `panel_visible_today` may be true if the user has SV open and set to show a URL from a previous day.
 
-### SV Panel
-- [ ] It should show a list of current http(s) tabs.
-- [ ] It should show a list of recently SVed tabs.
-- [ ] If !recently SVed tabs, only show the current tab section.
-- [ ] It should show a feedback button with the query params `ver={add-on-version}`, `ref=panel` and `rel={firefox-release}`.
-- [ ] The browser action icon should accept a fill value of `context-fill`
-- [ ] It should let the user toggle between mobile and desktop UI.
-- [ ] It should remember whether a site has been toggled to/from desktop before and render accordingly.
+## Onboarding
+- [ ] It should show onboarding to outline the basic usage of the feature (outline copy can be borrowed from [here](https://testpilot.firefox.com/experiments/side-view))
+- [ ] Users in `elective_onboarding` branch should only see onboarding on their first interaction with Side View
+- [ ] Users in the `default_onboarding` branch should see onboarding when the Side View shield study is installed
+- [ ] For users in the `elective_onboarding` branch, Side View should *only* be accessible from the browserAction until they are onboarded.
```

Comment:
We talked about intercepting first use for elective_onboarding, and then showing the onboarding in the sidebar, with the intended link in a query string or fragment. Then we'll direct the person to their intended destination at the end of onboarding (or if they just try again the second time will skip onboarding)